### PR TITLE
Fix showing newly added file-entry instantly

### PR DESCRIPTION
**Pull request by lucashabersaat**

### CODE REVIEW

File: frontend/app/models/file-entry.js [hunk 1]
```diff
@@ -10,12 +10,24 @@ export default class FileEntry extends Model {
...
+    let promise = this.file.upload(url, opts);
+    promise
+      .then( savedRecords => {
+        let data = JSON.parse(savedRecords.body).data
+        this.id = data.id;
+        this.filename = data.attributes.filename;
+      })
+      .catch(() => {
+        this.account = null;
+      });
```
Comment (lucashabersaat) [inline frontend/app/models/file-entry.js:29]: Didn't work in `handleSubmitError` in [file-entry.js](https://github.com/puzzle/cryptopus/blob/55a70697d667a38f3e7029c4b22678d6e5fa63ab/frontend/app/components/file-entry-form.js#L62).

File: frontend/app/components/account-show.js [hunk 2]
```diff
@@ -6,9 +6,14 @@ import { inject as service } from "@ember/service";
...
   @service store;
 
-  fileEntries = this.store.query("file-entry", {
-    accountId: this.args.account.id
-  });
+
+  constructor() {
+    super(...arguments);
+
+    this.fileEntries = this.store.query("file-entry", {
```
Comment (TheWalkingLeek) [inline frontend/app/components/account-show.js:13]: We don't need to assign this to a variable anymore

File: frontend/app/models/file-entry.js (hunk 1 above)
Comment (TheWalkingLeek) [reply frontend/app/models/file-entry.js:29]: hmm can you not directly set the account on the record inside file-entry-form? Not using the changeset
`this.record.account = null`

Comment (lucashabersaat) [reply frontend/app/models/file-entry.js]: Thanks for this suggestion, works.

**Pull request merged**